### PR TITLE
Fix Gazebo fixture errors

### PR DESCRIPTION
**Pull request by sharvenp**
## Fixes in this PR
- Gazebo fixture now loads and functions properly
- Migrated all Gazebo screens to use `defineComponent`

## Further work/comments/questions
- **Further Work:**
    - Migrate Vue2 Gazebo loading methods to Vue3 (~~TODO: Add issue to backlog~~ ✔)

## Steps to test
[Demo - Panel Party](http://ramp4-app.azureedge.net/demo/users/sharvenp/vue3-gazebo-fixture/host/index-e2e.html?script=panel-party)

1. Load Demo
2. Open the Gazebo panel and navigate through its buttons (don't forget to see the cat!)
3. Switch the language - the panel should update its text
    - Note that this will throw some warnings about language keys not found - this is expected.

### CODE REVIEW

**Comment by elsa-huang** [comment]
Just wondering why we're adding gazebo to `ramp-starter`?

**Comment by sharvenp** [comment]
> Just wondering why we're adding gazebo to `ramp-starter`?

Oh I added it for testing and forgot to remove it 😅
I'll push an update

File: packages/ramp-core/src/fixtures/gazebo/p1-screen-1.vue [hunk 1]
```diff
@@ -37,33 +37,34 @@
...
+    name: 'GazeboP1Screen1V',
+    data() {
+        return {
+            route: sync('panel/items@p1.route'),
+            pinned: sync('panel/pinned'),
+            url:
+                'https://i2.wp.com/freepngimages.com/wp-content/uploads/2017/08/wooden-garden-gazebo.png?w=860'
+        };
+    },
+    comnputed: {
```
Comment (elsa-huang) [inline packages/ramp-core/src/fixtures/gazebo/p1-screen-1.vue:54]:
```suggestion
    computed: {
```
spelling :(

File: packages/ramp-core/src/fixtures/gazebo/p1-screen-2.vue [hunk 2]
```diff
@@ -30,33 +26,38 @@
...
+
+export default defineComponent({
+    name: 'GazeboP1Scree2V',
+    data() {
+        return {
+            url:
+                'https://i2.wp.com/freepngimages.com/wp-content/uploads/2016/02/garden-shed-transparent-image-2.png?w=693'
+        };
+    },
+    comnputed: {
```
Comment (elsa-huang) [inline packages/ramp-core/src/fixtures/gazebo/p1-screen-2.vue:40]:
```suggestion
    computed: {
```
here too

**Review by elsa-huang — COMMENTED** [review]
Could you add TODOs where you changed the screen loading method to direct loading so that once we migrate the other loading methods we know to go back and change those again?

File: packages/ramp-core/src/fixtures/gazebo/p1-screen-1.vue (hunk 1 above)
Comment (sharvenp) [reply packages/ramp-core/src/fixtures/gazebo/p1-screen-1.vue]: Surprised it still worked 🤔

Donethanks

File: packages/ramp-core/src/fixtures/gazebo/p1-screen-2.vue (hunk 2 above)
Comment (sharvenp) [reply packages/ramp-core/src/fixtures/gazebo/p1-screen-2.vue]: Donethanks

**Review by elsa-huang — APPROVED** [review]

**Review by RyanCoulsonCA — APPROVED** [review]

**Review by yileifeng — APPROVED** [review]

**Pull request merged**